http2-tests/src/test/scala/org/apache/pekko/http/impl/engine/http2/HttpMessageRenderingSpec.scala

```diff
@@ -147,21 +145,39 @@ class HttpMessageRenderingSpec extends AnyWordSpec with Matchers {
       value1.exists(_._1 == "date") shouldBe false
     }
 
+    "handle empty trailer" in {
+      val config = ConfigFactory.load("reference.conf")
+      Try {
+        val rendering = new RequestRendering(ClientConnectionSettings(config), NoLogging)
+        rendering(HttpRequest().withAttributes(Map(AttributeKeys.trailer -> Trailer())))
+      }.isSuccess shouldBe true
+      Try {
+        val rendering = new ResponseRendering(ServerSettings(config), NoLogging, dateHeaderRendering)
+        rendering(
+          HttpResponse().withAttributes(
+            Map(
+              AttributeKeys.trailer -> Trailer(),
+              Http2.streamId -> 0)))
+      }.isSuccess shouldBe true
+    }
+
   }
 
-  private def renderClientHeaders(headers: immutable.Seq[HttpHeader], builder: VectorBuilder[(String, String)],
```

Comment:
please revert this change - it is probably here for Scala 2.12 support and there is no reason for this PR to be changing these methods anyway

Reply:
I think IntelliJ is deleting the Seq import, thinking its useless

Reply:
it's not useless - add it back

Reply:
yep sorry